sapi/fpm/fpm/events/port.c

```diff
@@ -138,8 +152,13 @@ static int fpm_event_port_wait(struct fpm_event_queue_s *queue, unsigned long in
 			continue;
 		}
 
+		ev = (struct fpm_event_s *)events[i].portev_user;
+
+		/* TODO: Escalate an error? */
+		fpm_event_port_add(ev);
```

Comment:
Is this part still needed? I remember that this might have not been port specific and possibly might be addressed by this commit https://github.com/php/php-src/commit/0ed6c3714087b254c49185568df96a31df76b2ce from @nikic if I understand the description of the patch (comment from Rainer in the bug report). There has been some other changes to signal handling from @plmnikulin that could possibly resolve this.

@psumbera can you try it without this change and confirm if it's still needed?